lessons/04-html-tables/index.html

```diff
@@ -0,0 +1,110 @@
+<html>
+    <head>
+        <title>Stanley Cup Champions</title>
+
+    </head>
+    <body>
+        <h1>Stanley Cup Champions</h1>
+        <h2>Every team that's won the Cup</h2>
+        <img src="https://nhl.bamcontent.com/images/photos/287516894/1024x576/cut.jpg" alt="">
+  <h3>Champions</h3>
+  <table>
+      <thead>
+          <tr>
+              <th>Year</th>
+              <th>Team</th>
+              <th>Coach</th>
+          </tr>
+      </thead>
+      <tbody>
+          <tr>
+              <td>2017</td>
+              <td>Pittsburgh Penguins</td>
+              <td>Mike Sullivan</td>
+          </tr>
+          <tr>
+              <td>2016</td>
+              <td>Pittsburg Penguins</td>
+              <td>Mike Sullivan</td>
+          </tr>
+          <tr>
+              <td>2015</td>
+              <td> Chicago Blackhawks</td>
+              <td> Joel Quenneville</td>
+          </tr>
+          <tr>
+              <td>2014</td>
+              <td>Los Angeles Kings</td>
+              <td> Darryl Sutter</td>
+          </tr>
+          <tr>
+              <td>2013</td>
+              <td>Chicago Blackhawks</td>
+              <td> Joel Quenneville</td>
+          </tr>
+          <tr>
+              <td>2012</td>
+              <td>Los Angeles Kings</td>
+              <td>Darryl Sutter</td>
+          </tr>
+          <tr>
+              <td>2011</td>
+              <td>Boston Bruins</td>
+              <td>Claude Julien</td>
+          </tr>
+          <tr>
+              <td>2010</td>
+              <td>Chicago Blackhawks</td>
+              <td> Joel Quenneville</td>
+          </tr>
+          <tr>
+              <td>2009</td>
+              <td>Pittsburgh Penguins</td>
+              <td>Dan Bylsma</td>
+          </tr>
+          <tr>
+              <td>2008</td>
+              <td></td>
+              <td></td>
+              <! ---empty tag--->
```

Comment:
Oops! Comments need to start with exactly `<!--`

In the above there is a space inbetween the exclamation mark and the dashes